docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
I think, you could use `FROM scratch` here.

We don't need a whole Linux system to use restic.